thundra/wrappers/wrapper_utils.py

```diff
@@ -125,3 +112,8 @@ def create_execution_context():
     transaction_id = str(uuid.uuid4())
     start_timestamp = int(time.time() * 1000)
     return ExecutionContext(transaction_id=transaction_id, start_timestamp=start_timestamp)
+
+
+def set_response_status(execution_context, response_status_code):
+    if response_status_code and not execution_context.invocation_data['userTags'].get('http.status_code'):
```

Comment:
Should we check whether `execution_context.invocation_data['userTags']` is exist?

Reply:
It is always initialized at that point but wouldn't hurt checking so I will update it